app/src/main/java/com/mapbox/maps/testapp/examples/annotation/SymbolActivity.kt

```diff
@@ -72,6 +72,12 @@ class SymbolActivity : AppCompatActivity() {
           val symbolOptions: SymbolOptions = SymbolOptions()
             .withPoint(Point.fromLngLat(AIRPORT_LONGITUDE, AIRPORT_LATITUDE))
             .withIconImage(it)
+            .withTextFont(
```

Comment:
Is it needed now? those values are default ones from `SymbolOptions` if I recall correctly

Reply:
This is a test to validate changing `text-font` for one symbol works.

Reply:
And this font stack is different from the default one.

Reply:
Yeah, I'm just a bit blind, sorry 🙈